policies/rudderc/Cargo.toml

```diff
@@ -11,9 +11,9 @@ license.workspace = true
 [dependencies]
 anyhow = "1"
 askama = "0.12"
-boon = "0.5.0"
+boon = "0.6.0"
 clap = { version = "4", features = ["derive"] }
-quick-xml = { version = "0.31.0", features = ["serialize"] }
+quick-xml = { version = "0.36.0", features = ["serialize"] }
```

Comment:
This seems tol break the metadata generation

Reply:
indeed (it is 0.32 with the changes in escaping)